docs/src/tutorials/supported_operations.md

```diff
@@ -99,14 +113,27 @@ sudo -u tedge touch /etc/tedge/operations/c8y/c8y_Restart
 
 Now the new operation will be automatically added to the list and the list will be sent to the cloud.
 
+To add a new operation to a child device create a new file in `/etc/tedge/operations/c8y/<child-device>` directory as below.
+
+```shell
+sudo -u tedge touch /etc/tedge/operations/c8y/<child-device>/c8y_Restart
+```
+Now the new operation will be automatically added to the list of child supported operations and will be sent to the cloud.
```

Comment:
Add a blank line as a separation from the previous paragraph.

```suggestion

Now the new operation will be automatically added to the list of child supported operations and will be sent to the cloud.
```

Reply:
fixed

Reply:
That commit seems missing ;-)